models/category-model.js

```diff
@@ -0,0 +1,12 @@
+const connection = require("../db/connection");
+
+exports.requestCategory = () => {
```

Comment:
Just a naming trick: For controllers you could use the http request names (ie. Get, Patch, Remove, Post), and then the PSQL syntax for the models (Select, Update, Delete, Insert).